python/nav/bin/sortedstats_cacher.py

```diff
@@ -48,11 +48,15 @@ def exit_if_running(pidfile):
         exit(1)
 
 
-if __name__ == '__main__':
+def main():
```

Comment:
You know me, I'm going to want this `main()` to come before the `run()` that it calls :)